CheatSheetVsCodePowerShell.md

```diff
@@ -0,0 +1,63 @@
+# VSCode Cheat Sheet – PowerShell
+## Specific PowerShell Commands
+
+|Command | Explanation |
+| --- | --- |
+|**CTRL + Shift + P (F1) PowerShell**|![alt text](./images/commandPallette.png "Use this to find commands specific for PowerShell")   | 
```

Comment:
I'd stick with a consistent casing of `Ctrl`, `Shift`, `Alt`.  We might want to have a footnote that describes how these keys map on macOS e.g. <kbd>Ctrl</kbd> to <kbd>Cmd</kbd>.